hypha/apply/funds/models/applications.py

```diff
@@ -232,6 +233,18 @@ class RoundBase(WorkflowStreamForm, SubmittableStreamForm):  # type: ignore
     )
     sealed = models.BooleanField(default=False)
 
+    def get_url(self) -> Optional[str]:
+        """Generates the live url, primarily used in the wagtail admin for the "view live" button.
+
```

Comment:
The only issue I had was when I went from my dashboard (`/dashboard/`) to the Wagtail admin dashboard (`/admin`) I got the error `RoundBase.get_url() got an unexpected keyword argument 'request'`. This seems to fix it just by adding the optional request kwarg. All looks good otherwise!

```suggestion
    def get_url(self, request: Optional[WSGIRequest]=None) -> Optional[str]:
        """Generates the live url, primarily used in the wagtail admin for the "view live" button.

        Args:
            request: The request that will be passed by Wagtail when loading the admin interface, currently unused.
```